app/tickers/[ticker]/CompanyInfo.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+import { useState } from 'react'
+
+export type CompanyInfoType = {
```

Comment:
No need to export this type I guess

Reply:
Also, since this type is a shorten version of `KeyStatsType`, you can just pick relevant props from `KeyStatsType`: https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys